exercises/allergies/uAllergyTests.pas

```diff
@@ -13,39 +13,39 @@   AllergyTests = class(TObject)
     procedure No_allergies_means_not_allergic;
 
```

Comment:
This first test should also have the "comment this out to run this test" but it should be commented out.